### DISCUSSION TITLE
Questions Re: Uploading Dataset with Annotations

### DISCUSSION THREAD

**Discussion by ddseddse** [cvat-ai/cvat#4170] [Q&A] [answered]
I've installed CVAT following the instructions [here](https://openvinotoolkit.github.io/cvat/docs/administration/basics/installation/) and it seems to be working flawlessly. This is t he version installed:
![image](https://user-images.githubusercontent.com/32875465/149373743-5bd169f5-c936-4df8-a2df-b2910c819c5c.png)

Now I want to import a PASCAL VOC format dataset. My first issue is when following the instructions [here](https://openvinotoolkit.github.io/cvat/docs/manual/advanced/export-import-datasets/), but in my Projects screen I do not get an "Import Dataset"option (or a "Backup Project" option for that matter). Here is what I see:
![image](https://user-images.githubusercontent.com/32875465/149374672-0f07a8db-a918-4d49-b78b-cd9d532b037f.png)

What do I need to do to get the Import option to show up?

Also, a second question. Under Tasks I have an option to import annotations and that leads to a file open dialog that is looking for a .zip file. What would the exacxt contents of that .zip file be for this purpose? 

Thanks!

Dan

**Comment by ddseddse** [comment, marked as answer]
The answer to Question #2 appears to be "zip up all the annotation XML files together and upload". That works fine. I guess I should have tried that first.

I would still appreciate any help on answering question #1 above (how to get the Import Dataset option to appear).

Thanks.

-Dan

**Reply by azhavoro** [reply]
Hi, you need to build images locally from develop branch with `docker-compose -f docker-compose.yml -f docker-compose.dev.yml up -d --build`

**Comment by cirobr** [comment]
Cheers,

I am trying to import the "Detection_and_localization" dataset from [this link](https://data.bris.ac.uk/data/dataset/4vnrca7qw1642qlwxjadp87h7). To the best of my knowledge, it is Pascal VOC compatible.

I've managed to upload all images to CVAT. I've also created a ZIP file with all corresponding XML annotation files. When trying to upload through Menu > Upload annotations > Import Format = Pascal VOC I get the error message as in the below picture:
![2023-03-28 (1)](https://user-images.githubusercontent.com/37887021/228299481-aed02616-769d-4ee8-9e56-fa78bbd7486b.png)

The corresponding annotation file follows:
```
<annotation verified="no">
  <folder>block_001</folder>
  <filename>image_0000026_2020-02-05_13-21-21</filename>
  <path>D:\Work\Data\RGBDCows2020\blocks\block_001\image_0000026_2020-02-05_13-21-21.jpg</path>
  <source>
    <database>Unknown</database>
  </source>
  <size>
    <width>1280</width>
    <height>720</height>
    <depth>3</depth>
  </size>
  <segmented>0</segmented>
  <object>
    <type>bndbox</type>
    <name>head</name>
    <pose>Unspecified</pose>
    <truncated>0</truncated>
    <difficult>0</difficult>
    <bndbox>
      <xmin>672</xmin>
      <ymin>166</ymin>
      <xmax>672</xmax>
      <ymax>166</ymax>
    </bndbox>
  </object>
  <object>
    <type>robndbox</type>
    <name>cow</name>
    <pose>Unspecified</pose>
    <truncated>0</truncated>
    <difficult>0</difficult>
    <robndbox>
      <cx>641.9505</cx>
      <cy>163.3515</cy>
      <w>493.1009</w>
      <h>237.9842</h>
      <angle>0.15</angle>
    </robndbox>
  </object>
</annotation>
```

Appreciate any advise on how to complete the task.

Thanks.
